components/Investors.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+
+const investors = [
+  'true ventures logo.png',
```

Comment:
@atepem would be great if you could rename all this images so that they don't have spaces, potentially can cause problems in different OSes with launching/building